app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -210,7 +208,7 @@ class RootActivity : AppCompatActivity(), RootScreen {
     private fun setupTimePicker() {
         ivyContext.onShowTimePicker = { initialTime,
                                         onTimePicked ->
-            val nowLocal = initialTime ?: timeNowLocal().toLocalTime()
+            val nowLocal = initialTime ?: timeProvider.localTimeNow()
             val is24Hour = android.text.format.DateFormat.is24HourFormat(this)
```

Comment:
This date-time picker should be deleted and fully migrated to `DeviceDateTimePicker`